datafusion/build.rs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+fn main() {
+    let version = rustc_version::version().unwrap();
+    println!("cargo:rustc-env=RUSTC_VERSION={}", version);
+}
```

Comment:
not sure this is something we'd add for datafusion

Reply:
Add build.rs  file  can help me easy to get the rustc_version compiled by the project .
Of course, I can modify ` pub static RUSTC_VERSION: &str = env! ("RUSTC_VERSION");`  In `datafusion / src / plugin / mod.rs`  Through `rustc_version::version().unwrap()` to get the rustc_version at compile time.